Aliucord/build.gradle

```
@@ -56,8 +56,6 @@ android.libraryVariants.all { variant ->
             source = compiler.source
             classpath = files(android.bootClasspath.join(File.pathSeparator))
             classpath += compiler.classpath
-
-            exclude 'com/discord/app/**'
```

```suggestion

            exclude 'com/discord/app/**'
```
`com/discord/app/App$a` still exists in Aliucord as backup